cheevos-new/fixup.c

```diff
@@ -260,10 +274,10 @@ const uint8_t* rcheevos_patch_address(unsigned address, int console)
 
          core_get_memory(&meminfo);
 
-         if (addr < meminfo.size)
+         if (address < meminfo.size)
          {
             pointer = meminfo.data;
-            address = addr;
+            address = address;
```

Comment:
does this `address = address` have any purpose?

Reply:
Nope. It's leftover from the renaming. I'll clean it up.

before:
1) copy `address` into `addr`
2) update `addr` several times
3) if successful, copy `addr` back into `address`
  otherwise, leave `address` unmodified for error message.

after:
1) copy `address` into `original_address` (above)
2) update `address` several times
3) if successful, copy `address` back into `address` (redundant)
  otherwise, use `original_address` for error message.